concent_api/core/tasks.py

```diff
@@ -114,6 +146,18 @@ def upload_finished(subtask_id: str) -> None:
         )
     # If it's one of the states that can precede verification, report an error. Processing ends here.
     else:
+        delete_deposit_claim(
+            subtask_id=subtask_id,
+            concent_use_case=ConcentUseCase.ADDITIONAL_VERIFICATION,
```

Comment:
Just like above. Something's wrong. This case should never happen unless there's a bug in the Conductor or Verifier and one of them has sent a notification even though there's no verification in progress. The claims should not even exist. Just leave them as is for later analysis by the admin.

Reply:
:+1: done